src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -43,7 +44,21 @@
         Recipe recipe = new Recipe(new Name("Aglio Olio"), new CompletionTime(5), new ServingSize(1),
                 ingredients, steps, getTagSet("Western", "Italian"));
 
-        return new Recipe[] {recipe};
+        //Nasi Lemak
+        Ingredient i1 = new Ingredient("fish", 10);
+        Ingredient i2 = new Ingredient("rice", 1, "- s cook the rice");
```

Comment:
This seems incorrect, Ingredient quantifier should not contain a step

```suggestion
        Ingredient i2 = new Ingredient("rice", 1, null);
```